File: cvahn_lap11_demboiso.py
def divisible_numbers(n, k, m):
    return len([i for i in range(1, n+1) if i % k == 0 or i % m == 0])

# Read the input file
with open('Demboi.inp', 'r') as f:
    k, m, n = map(int, f.readline().split())

# Calculate the result
result = divisible_numbers(n, k, m)

# Write the result to the output file
with open('Demboi.out', 'w') as f:
    f.write(str(len(result)))

File: cvahn_lap11_demboiso.cpp
#include <fstream>
#include <algorithm>

int main() {
    int k, m, n;
    std::ifstream input("demboi.inp");
    input >> k >> m >> n;
    input.close();

    int count = 0;
    for (int i = std::min(k, m); i <= n; ++i) {
        if (i % k == 0 || i % m == 0) {
            ++count;
        }
    }

    std::ofstream output("demboi.out");
    output << count;
    output.close();

    return 0;
}

